include/zephyr/drivers/sensor.h

```diff
@@ -186,6 +186,8 @@ enum sensor_channel {
 	SENSOR_CHAN_GAUGE_DESIRED_VOLTAGE,
 	/** Desired charging current in mA */
 	SENSOR_CHAN_GAUGE_DESIRED_CHARGING_CURRENT,
+	/** Fan faults, one bit represents one channel */
+	SENSOR_CHAN_FAN_FAULT,
```

Comment:
This should be in a private header, rather than the common one.

Reply:
Could you please explain why? Because the fan speed (SENSOR_CHAN_RPM) is also used from this global header.

Reply:
SENSOR_CHAN_RPM is generic and can be used by multiple sensors, whereas SENSOR_CHAN_FAN_FAULT seems rather specific to this device.

Device specific channels are implemented in a separate header, e.g:
https://github.com/zephyrproject-rtos/zephyr/blob/1a3bb3fd5c87572da19a6a54063e4d23cca2ac03/include/zephyr/drivers/sensor/npm1300_charger.h

Reply:
done